nemo/collections/tts/parts/tacotron2.py

```diff
@@ -488,7 +488,7 @@ def infer(self, memory, memory_lengths):
             alignments += [alignment]
 
             if len(mel_outputs) == self.max_decoder_steps:
-                logging.warning("Reached max decoder steps")
+                nemo.logging.warning(f"Reached max decoder steps {self.max_decoder_steps}")
```

Comment:
please switch back to logging.warning. Also please don't use format strings in logging messages.